data_access.go

```diff
@@ -156,7 +168,31 @@ func (s *DataAccessSyncer) importDataAccess(config *data_access.DataAccessSyncCo
 			}
 		}
 
-		err = fileCreator.AddAccessProvider([]dap.AccessProvider{da})
+		// copy AccessObjects from this role to all roles that have a GRANT on this one
+		for _, grantee := range grantOfEntities {
+			if grantee.GrantedTo == "ROLE" {
+				if _, f := accessProviderMap[grantee.GranteeName]; !f {
+					accessProviderMap[grantee.GranteeName] = &dap.AccessProvider{
+						ExternalId: grantee.GranteeName,
+						Name:       grantee.GranteeName,
+					}
+				}
+				granteeDa := accessProviderMap[grantee.GranteeName]
+				granteeDa.AccessObjects = append(granteeDa.AccessObjects, da.AccessObjects...)
+				logger.Info(fmt.Sprintf("Adding AccessObjects for role %s to grantee %s", roleEntity.Name, granteeDa.Name))
+			}
+		}
+	}
+
+	for _, da := range accessProviderMap {
+		if isSnowFlakeDefaultRole(da.Name) {
```

Comment:
This will need to also include the imported roles/Groups from Okta (or something else). Can be detected through the 'owner' column. We should probably have a config option for that as well (which user/role acts as the importer role, like we already have for the user ignore)

Reply:
Indeed, I also wanted to mention the option of it being configurable. If you follow the tutorial, the role/owner is `OKTA_PROVISIONER`, but you can of course enter what you want in there.